urllib3/connection.py

```diff
@@ -162,6 +162,37 @@ def connect(self):
         conn = self._new_conn()
         self._prepare_conn(conn)
 
+    def request_chunked(self, method, url, body=None, headers={}):
+        """
+        Alternative to the common request method, which sends the
+        body with chunked encoding and not as one block
+        """
+        header_names = set(k.lower() for k in headers)
+        self.putrequest(method, url, skip_accept_encoding=True)
```

Comment:
It's not skipping the header, it's skipping the auto-insertion of an 'identity' accept_encoding header. This is the line from httplib:

```
            if not skip_accept_encoding:
                self.putheader('Accept-Encoding', 'identity')
```